virt_lightning/__init__.py

```diff
@@ -87,10 +90,17 @@ def get_free_ipv4(self):
                 self._last_free_ipv4 = interface
                 return interface
 
+    def get_storage_dir(self):
+        xml = self.pool.XMLDesc(0)
+        root = ET.fromstring(xml)
+        disk_source = root.find("./target/path")
+        return disk_source.text
+
 
 class LibvirtDomain:
-    def __init__(self, dom):
+    def __init__(self, dom, hv=None):
         self.dom = dom
+        self.hv = hv  # TODO: Cross reference!
```

Comment:
Where is initialization without `hv` constructor init?

Reply:
We don't always need it. BTW, I prepared a patch to drop the cross dependency between the two class.